terraform/shared/shared.tf.json.template.py

```diff
@@ -378,9 +382,43 @@ def paren(s: str) -> str:
                 }
             }
         },
-        **(
-            {
-                'aws_cloudformation_stack': {
+        'aws_cloudformation_stack': {
+            # Images whose short name begins with '_' are only used outside the
+            # security boundary, so vulnerabilities that are detected within
+            # them do not need to be addressed with the same urgency.
+            #
+            # Using CF stack because the AWS provider does not support
+            # Inspector filters and the AWSCC provider fails due to
+            # https://github.com/hashicorp/terraform-provider-awscc/issues/1364
+            'inspector_filters': {
+                'name': config.qualified_resource_name('inspectorfilters'),
+                'template_body': json.dumps({
+                    'AWSTemplateFormatVersion': '2010-09-09',
+                    'Description': 'Create suppression rules for select Docker images in AWS Inspector',
+                    'Resources': {
+                        # Must be alphanumeric
+                        hashlib.md5(short_name.encode()).hexdigest(): {
+                            'Type': 'AWS::InspectorV2::Filter',
+                            'Properties': {
+                                'Name': 'exclude_image' + short_name,
+                                'FilterAction': 'SUPPRESS',
+                                'FilterCriteria': {
+                                    'EcrImageRepositoryName': [
+                                        {
+                                            'Comparison': 'EQUALS',
+                                            'Value': ImageRef.parse(full_name).name
+                                        }
+                                    ]
+                                }
+                            }
+                        }
+                        for short_name, full_name in config.docker_images.items()
```

Comment:
Please use `azul.docker.images` to obtain `ImageRef` instances instead of parsing them here. You'll need to convert that variable to a dictionary to retain the short name.